src/TruncatedAccount/index.js

```diff
@@ -14,7 +14,7 @@ const TruncatedAccount = ({ name, lastFour }) => (
         -
       </span>
     )}
-    {lastFour && <span>{lastFour}</span>}
+    {lastFour && <span className="whiteSpace--singleLine">{lastFour}</span>}
```

Comment:
🎉 great fix, honestly - I didn't consider just applying this again within the span